### DISCUSSION TITLE
How to, prevent mozc from suggesting conversion of 'sa' / さ to Kanji?

### DISCUSSION THREAD

**Discussion by octimis** [google/mozc#1094] [Q&A]
I'm aware this can be 'temporarily' avoided by pressing enter key instead of space key. But I'm if this 'suggestion' can be disable for specific characters, instead of inverting/swapping the key settings (set space to enter and set enter to space).

**Comment by hiroyuki-komatsu** [comment]
Would you please share your use case why you want to do that?
Steps to reproduce and a screen record are also helpful for us to understand your situation.
